### DISCUSSION TITLE
Option to hide only contribution activities in Profile

### DISCUSSION THREAD

**Discussion by lfom** [community/community#43304] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello!

Please consider adding an option to hide only recent activities in the profile. Currently, the only option is "Make profile private and hide activity", which basically hides everything in your profile. I think an option to only hide the recent activity would be very useful, keeping the profile public with all info but hiding only the user activity. Thank you.

Regards

**Comment by Ankush11903** [comment]
okay.
so , that's why your account is private.

**Comment by radar-blip** [comment]
Maybe you would find this setting useful: 

![hideactivity](https://user-images.githubusercontent.com/39196767/210943945-2eb19583-3011-424a-9cef-09a0067860b8.png)

**Reply by lfom** [reply]
Thanks, but this only hides the overview below the Contributions table (which was already disabled in my case anyway), and not the Contribution activity itself. What I suggested was keeping the profile visible, but hide only the user activity (or the whole Contributions part only, if hiding only the activity is not possible or too complicated).
